nl_server/config_reader.py

```diff
@@ -98,7 +98,7 @@ def merge_vertex_ai_configs(m: ModelConfig,
   return VertexAIModelConfig(**merged)
 
 
-def read_catalog() -> Catalog:
+def read_catalog(catalog_dict: Dict = None) -> Catalog:
```

Comment:
I like this change and we should eventually make this 

`read_catalog(catalog_paths: [] = DEFAULT_PATHS, catalog_dict: Dict = None)`

Right now the custom DC path prevents the path to be constant, and really that should be passed in as the catalog_dict string value. 

Can you add a TODO for this to remind me of the change.

Reply:
added TODO